File: Programming Basics with JS/T04 - Nested Conditional Statements/lecture/04. Personal Titles.js
function ageAndSex(input) {
    let age = Number(input[0]);
    let sex = input[1];

    if (sex === "f") {
        if (age < 16) {
            console.log("Miss");
        } else {
            console.log("Ms.")
        }

    } else if (sex === 'm'){
        if (age < 16) {
            console.log('Master');
        } else {
            console.log('Mr.');
        }
    }
}

File: Programming Basics with Go/T04 - Conditional Statements (Advanced)/lecture/04. Personal Titles.go
package main

import "fmt"

func main() {

	var age float32
	fmt.Scanln(&age)

	var gender string
	fmt.Scanln(&gender)

	//switch  {} -> серия от различни проверки

	//мъж -> < 16 или >= 16
	//жена -> < 16 или >= 16

	if gender == "m" {
		//мъж
		//проверка за възрастта -> < 16; >= 16
		switch {
			case age < 16:
				fmt.Print("Master")
			case age >= 16:
				fmt.Print("Mr.")
		}
	} else if gender == "f" {
		//жена
		//проверка за възрастта
		if age < 16 {
			fmt.Print("Miss")
		} else {
			// >= 16
			fmt.Print("Ms.")
		}
	}



}